packages/test/test-end-to-end-tests/src/test/stashedOps.spec.ts

```diff
@@ -1076,6 +1076,51 @@ describeNoCompat("stashed ops", (getTestObjectProvider) => {
 		);
 	});
 
+	it("not expired stashed blobs", async function () {
+		const container = await loadOffline(provider, { url });
+		const dataStore = await requestFluidObject<ITestFluidObject>(
+			container.container,
+			"default",
+		);
+		const map = await dataStore.getSharedObject<SharedMap>(mapId);
+
+		// Call uploadBlob() while offline to get local ID handle, and generate an op referencing it
+		const handle = await dataStore.runtime.uploadBlob(
+			stringToBuffer("blob contents 1", "utf8"),
+		);
+		map.set("blob handle 1", handle);
+
+		container.connect();
+		await waitForContainerConnection(container.container, true);
```

Comment:
I'm not confident this will test what we want. The BlobAttach op could round-trip before we stash due to async timing, in which case we will get no stashed blobs. It would be good to assert that stashedChanges has some blobs in it.

Also, waiting for container connection won't guarantee the blobs have been uploaded. If we wait for "connected" event on container runtime, that should mean any offline blobs have been uploaded (since runtime delays transitioning to "connected" state while it uploads offline blobs).

Reply:
I added a couple of asserts here, verifying that blob was stashed and uploaded before closing. Not sure if by your explanation this would lead to flakiness though, but I've tested it a couple dozens of times and seems consistent

Reply:
Looks like it's failing on t9s because t9s doesn't respond with TTL? I think waiting for runtime "connected" event would work, and asserting the minTTL wouldn't be necessary then.

Reply:
Just added skipping if driver is not local or odsp

Reply:
Not sure I like that. Nothing the test covers is unsupported on other drivers, why are we skipping it? Skipping the assert itself would be better, but I still don't see a reason not to wait for runtime "connected" instead of container "connected"

Reply:
The whole test does not have sense if there is no TTL so I though skipping the test altogether would be fine.